src/metabase/api/macros/defendpoint/open_api.clj

```diff
@@ -130,35 +131,58 @@
                   schema))
               (mc/children schema))))))
 
+(def ^:private default-response-schema
+  "Default response schema for OpenAPI endpoints. This is used when the endpoint does not specify a response schema."
+  {"2XX" {:description "Successful response"}
+   "4XX" {:description "Client error response"}
+   "5XX" {:description "Server error response"}})
```

Comment:
Even though it doesn't really say much, It's a fallthru, it's correct, and it avoids the error: 

<img width="719" height="407" alt="Image" src="https://github.com/user-attachments/assets/3e7367ba-7c4b-44d7-a380-768366e1c597" />